app/Event.php

```diff
@@ -83,4 +84,12 @@ public function updateEventById($eventArr, $id)
     }
     #endregion
 
+    #region Hide Event
+    public function hideEvent()
+    {
+        return $this->where('published_date', '>=', Carbon::now())
```

Comment:
shouldn't this be `<=`, because we can only see events after they are published, not before they are published